portality/bll/services/article.py

```diff
@@ -50,10 +50,11 @@ def batch_create_articles(self, articles, account, duplicate_check=True, merge_d
         all_unowned = set()
         all_unmatched = set()
 
+        ids = []
 
         for article in articles:
             try:
-                result = self.create_article(article, account,
+                result, id = self.create_article(article, account,
```

Comment:
See comment on below about the `batch_create_article` method.  I don't think this change is necessary, but if it is I'd want to talk through the implementation of a change to the BLL's method signature, as I don't think this is the cleanest approach.